app/helpers/schema_helpers.py

```diff
@@ -8,11 +8,8 @@
 from app.questionnaire.questionnaire_schema import QuestionnaireSchema
 from app.utilities.schema import load_schema_from_metadata
 
-T = TypeVar("T")
-P = ParamSpec("P")
 
-
-def with_schema(
+def with_schema[**P, T](
```

Comment:
According to the docs, The preferred way to construct a type variable is via the dedicated syntax for [generic functions](https://docs.python.org/3/reference/compound_stmts.html#generic-functions), [generic classes](https://docs.python.org/3/reference/compound_stmts.html#generic-classes), and [generic type aliases](https://docs.python.org/3/reference/compound_stmts.html#generic-type-aliases). https://docs.python.org/3/library/typing.html#typing.TypeVar